strabo/templates/public/popup.html

```diff
@@ -1,31 +1,27 @@
 <!--
 Organzes the popup
 -->
-
 <div class="popup" hidden>
     <div class="popup-foreground horizontal-center vertical-center">
-        <div class="top-section">
-            <h2 id="ip_title" class="col-xs-10">
-                <!--javascrip puts title item in here -->
+        <div id="button-section">
+            <label class="close-button" onclick="hide_popup();">&times;</label>
```

Comment:
Can we add a comment that states that `&times;` is the HTML entity we are using for the close button? that confused me until I looked it up, and it may help people who look at our code later